src/apps/alerts/crud/alert.py

```diff
@@ -76,22 +76,19 @@ async def get_by_applet_id(
         # Get alert from the database
         query: Query = select(
             self.schema_class,
-            AlertConfigSchema.alert_message.label("alert_message"),
             AppletSchema.display_name.label("applet_name"),
             UserAppletAccessSchema.meta.label("meta"),
         )
         query = query.where(self.schema_class.applet_id == applet_id)
-        query = query.join(
-            AlertConfigSchema.alert_message,
-            AlertConfigSchema.id == self.schema_class.alert_config_id,
-        )
         query = query.join(
             AppletSchema.display_name,
             AppletSchema.id == self.schema_class.applet_id,
         )
         query = query.join(
             UserAppletAccessSchema.meta,
```

Comment:
Could you use `join` in this format `.join(Table, Condition)` to follow one structure